src/fieri/app/models/license_worker.rb

```diff
@@ -0,0 +1,43 @@
+require 'sidekiq'
+require 'net/http'
+
+class LicenseWorker
+  include ::Sidekiq::Worker
+
+  # Acceptable licenses were determined and documented in
+  # https://github.com/chef-cookbooks/cookbook-quality-metrics/blob/master/quality-metrics/qm-003-license.md
+  ACCEPTABLE_LICENSES = ['Apache 2.0', 'MIT', 'GNU Public License 2.0', 'GNU Public License 3.0'].freeze
```

Comment:
@nellshamrell and I discussed and are OK with moving ahead with this list while we have the metric flipped to admin-only. But we should decide on a canonical list of license identifiers to use before making this metric public.